src/config/mod.rs

```diff
@@ -1986,6 +1986,19 @@ impl RaftEngineConfig {
         Ok(())
     }
 
+    fn optimize_for(&mut self, raft_store: &RaftstoreConfig, raft_kv_v2: bool) {
+        if !raft_kv_v2 {
+            let cur_batch_compression_thd = self.config().batch_compression_threshold;
+            // As the async-io is enabled by default (raftstore.store_io_pool_size == 1),
+            // testing records shows that using 4kb as the default value can achieve
+            // better performance and reduce the IO overhead.
+            self.mut_config().batch_compression_threshold = RaftEngineReadableSize(std::cmp::min(
```

Comment:
Shall we only change this config when it is the previous default value? I concern that it may let to some unexpected value if use explicted set `store_io_pool_size ` and `batch_compression_threshold `. I think it's better change to logic to:
```
if self.config().batch_compression_threshold == 8kb && raft_store.store_io_pool_size > 0 {
    self.mut_config().batch_compression_threshold = 4kb;
}
```

Reply:
Thx for reminding, I've forgotten to submit this part.